.maestro/51-threads.yaml

```diff
@@ -0,0 +1,202 @@
+appId: io.tlon.groups
+---
+- runFlow: subflows/launch.yaml
+- runFlow: subflows/tlon-login.yaml
+- assertVisible: 'Home'
+# Tap on the "+" button
```

Comment:
might be worth sub-flowing the group creation

Reply:
Possibly, we do it enough